autotm/utils.py

```diff
@@ -105,13 +104,14 @@ def parallelize_dataframe(df: pd.DataFrame, func, n_cores, return_type="df", **k
 
 
 def make_log_config_dict(
-    filename: str = "/var/log/tm-alg.txt", uid: Optional[str] = None
+        filename: str = "/var/log/tm-alg.txt", uid: Optional[str] = None,
+        quiet: bool = False
 ) -> Dict[str, Any]:
     if filename is not None:
         if uid:
             dirname = os.path.dirname(filename)
             file, ext = os.path.splitext(os.path.basename(filename))
-            log_filename = os.path.join(dirname, f"{file}-{uid}.{ext}")
+            log_filename = os.path.join(dirname, f"{file}-{uid}{ext}")
```

Comment:
why removing the dot before the extension?

Reply:
Because `ext` already includes a dot